config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
   root controller: :index, action: :index
-  resources :articles, only: [:new, :create, :show, :destroy]
+  resources :articles, only: [:new, :create, :show, :destroy, :update, :edit]
```

Comment:
At this stage you can probably remove the `only: ` setting. You are using all CRUD and helper routes.